<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1230</width>
    <height>850</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1230</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1230</width>
    <height>850</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="settings_frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>250</width>
      <height>650</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>650</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>650</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QPushButton" name="startAcquisitionBtn">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>510</y>
       <width>220</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>Start Acquisition</string>
     </property>
    </widget>
    <widget class="QPushButton" name="liveScanBtn">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>270</y>
       <width>220</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>Live Scan</string>
     </property>
    </widget>
    <widget class="QPushButton" name="settingsBtn">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>350</y>
       <width>220</width>
       <height>140</height>
      </rect>
     </property>
     <property name="text">
      <string>Save Directory,
Format, Name,
Autoupload</string>
     </property>
    </widget>
    <widget class="QPushButton" name="advancedSetupBtn">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>190</y>
       <width>220</width>
       <height>60</height>
      </rect>
     </property>
     <property name="text">
      <string>Advanced Setup</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QFrame" name="frame_3">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>20</y>
       <width>220</width>
       <height>161</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="formLayoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>201</width>
        <height>141</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="ledIntensityLabel">
         <property name="text">
          <string>LED intensity (%)</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="frameRateFPSLabel">
         <property name="text">
          <string>Frame Rate (FPS)</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="durationSLabel">
         <property name="text">
          <string>Duration (s)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="ledIntensityEdit">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="frameRateEdit">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="maximum">
          <number>500</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="durationEdit">
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::NoButtons</enum>
         </property>
         <property name="maximum">
          <number>10000</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <zorder>frame_3</zorder>
    <zorder>startAcquisitionBtn</zorder>
    <zorder>liveScanBtn</zorder>
    <zorder>settingsBtn</zorder>
    <zorder>advancedSetupBtn</zorder>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>960</x>
      <y>60</y>
      <width>250</width>
      <height>650</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>250</width>
      <height>650</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>250</width>
      <height>650</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QLabel" name="platemap">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>20</y>
       <width>215</width>
       <height>215</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>Plate Map</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="readout">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>260</y>
       <width>215</width>
       <height>215</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="text">
      <string>Temperature and
environment
readouts</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string>Choose the plate format and profile: 2D vs Mantarray and each will have their own subgroups of 24, 48, 96, (maybe 384-well later).</string>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="indent">
     <number>10</number>
    </property>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>720</y>
      <width>650</width>
      <height>100</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>650</width>
       <height>100</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="image">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>60</y>
      <width>650</width>
      <height>650</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
